components/KebabPopUp.tsx

```diff
@@ -23,6 +23,7 @@ export const KebabPopUp: React.FC<KebabPopUpProps> = props => {
         name={props.iconName}
         type={props.iconType}
         color={Theme.Colors.GrayIcon}
+        size={16}
```

Comment:
can we get this number from theme and reuse instead of hardcoding?

Reply:
Moved to themes